src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -130,34 +130,6 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
   }
 
   renderZendeskScript() {
-    const {
```

Comment:
I removed all this logic since 1) auctions don't have price listings so the button would never show up and 2) the other checks apply to ecommerce artworks, however, we are now showing the ZD icon on those pages as well so I think its safe to clean this up here.

Reply:
I think we would still want to have the thresholds on the artwork pages thought right? Unless MSO change the requirements?

Reply:
We removed the thresholds on BNMO works in the checkout flow,  it like we should be consistent and do the same thing on the artwork page? Also auction works don't have artwork prices so we'd do an early return here without this change. I can follow up with someone on MSO to see if this change is ok though.